GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -125,7 +125,12 @@ private void PopulateChoices()
                 proteaseComboBox.Items.Add(protease);
             }
 
-            proteaseComboBox.SelectedIndex = 12;
+            proteaseComboBox.SelectedIndex = 1;
```

Comment:
I'm not really a fan of this either... can we just do `proteaseComboBox.SelectedItem = "trypsin";` ? any issues with that?